### PR TITLE
Added separate alarm for ok actions where description looks fine for in alarm and ok alarm  #minor

### PR DESCRIPTION
**Pull request by mayank0202**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -1,47 +1,69 @@
...
 }
 
-resource "aws_cloudwatch_metric_alarm" "this" {
-  alarm_name          = "${var.alarm_prefix}-${var.domain_name}${var.resource_path}"
+resource "aws_cloudwatch_metric_alarm" "alarm_breaching" {
+  alarm_name          = "${var.alarm_prefix}-${var.domain_name}${var.resource_path}-Breaching"
   namespace           = "AWS/Route53"
   metric_name         = "HealthCheckStatus"
-  comparison_operator = "LessThanThreshold"
+  comparison_operator = "LessThanOrEqualToThreshold"
```
Comment (vijay-stephen) [inline main.tf:24]: create a variable and default the value

File: main.tf [hunk 2]
```diff
@@ -1,47 +1,69 @@
...
   evaluation_periods  = var.evaluation_periods
   period              = var.period
-  statistic           = "Minimum"
+  statistic           = "Average"
```
Comment (vijay-stephen) [inline main.tf:27]: create a variable and default the value

File: main.tf [hunk 3]
```diff
@@ -1,47 +1,69 @@
...
-  ok_actions                = [aws_sns_topic.this.arn]
   insufficient_data_actions = []
   treat_missing_data        = "breaching"
   depends_on = [
     aws_route53_health_check.this
   ]
+  tags = var.tags
+}
 
+resource "aws_cloudwatch_metric_alarm" "alarm_ok" {
```
Comment (vijay-stephen) [inline main.tf:43]: Create a flag to enable disable

Comment (vijay-stephen) [reply main.tf:43]: Also rename the resource name as just `ok` instead of `alarm_ok`

**Review by vijay-stephen — CHANGES_REQUESTED** [review]

File: main.tf (hunk 3 above)
Comment (mayank0202) [reply main.tf]: Done

**Review by vijay-stephen — APPROVED** [review]

**Pull request merged**